packages/istanbul-reports/lib/html/assets/base.css

```diff
@@ -222,3 +226,48 @@ pre.prettyprint {
 .footer, .push {
   height: 48px;
 }
+
+
+@media (prefers-color-scheme: dark) {
+  body {
+    color:#ddd;
```

Comment:
Lets also set `background:#222;` here.  This will set the stage for everything else.

Reply:
Fixed in 61082b174323d810243a24e45d7cff3e721cd93a